pkg/upgrade/upgrademanager/manager.go

```diff
@@ -756,7 +756,7 @@ SELECT id, status
 			payload,
       false -- emit_defaults
 		) AS pl
-	FROM system.jobs
+	FROM crdb_internal.system_jobs
```

Comment:
Do we have a concise explanation on who (what packages) should be allowed to access system.jobs?  Is it only underlying jobs package (and maybe tests)? Should there be a lint to warn about
system.jobs usage instead of crdb_internal.system_jobs?